uk.co.compucorp.civicrm.hrcore/gulp/utils.js

```diff
@@ -237,6 +240,25 @@ function getExtensionPath (name) {
   return extensionsPathCache[extension];
 }
 
+/**
+ * Returns the URL path for the given extension. The URL path is cached for
+ * optimization reasons.
+ *
+ * @param {String} extensionName the name of the extension to query for the URL path
+ * @return {String}
+ */
+function getExtensionUrlPath (extensionName) {
```

Comment:
like for the ext path function, if the name is not passed it should use the current extension by default